### PR TITLE
Fix#90 [IMP] replace only button. Do not use 'form' tag. Use 4 buttons inste…

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
…ad of radio.

### CODE REVIEW

**Comment by yelizariev** [comment]
FIX website_sale_clear_cart to make it compatible with website_sale_checkout_store

**Comment by yelizariev** [comment]
Fix all lint erros for **website_sale_checkout_store**
 https://travis-ci.org/it-projects-llc/website-addons/jobs/155877359

**Comment by yelizariev** [comment]
add     

    width: 100%;

to buttons

File: website_sale_checkout_store/__openerp__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 {
     'name': """Pickup and pay at store""",
-    'summary': """Simplify checkout process by excluding shipping and\or payment information""",
+    'summary': """Simplify checkout process by excluding shipping and|or payment information""",
```
Comment (yelizariev) [inline website_sale_checkout_store/__openerp__.py]: +    'summary': """Simplify checkout process by excluding shipping and/or payment information""",

?

Comment (ilyasProgrammer) [inline website_sale_checkout_store/__openerp__.py]: lint или pep ругался

**Pull request merged**